README.md

```diff
@@ -245,3 +246,7 @@ First, please read the [contributing docs](CONTRIBUTING.md). Generally, you will
 ## Adding new sbatch scripts
 
 You can add more sbatch scripts by putting them in the sbatches directory.
+=======
```

Comment:
This looks like it’s unintended (at the top too).

Reply:
Should be fixed on the latest commit 
38bbbff